src/app.js

```diff
@@ -0,0 +1,23 @@
+import { div } from '@cycle/dom'
+import xs from 'xstream'
+import World from './components/world'
+import Cyclejs from './components/cyclejs'
+
+export default ({ DOM$ }) => {
+  /*
```

Comment:
Commentary is huge for nothing and it's alone.
Delete it or comment all file with `// world` style comentaries